src/boot/sentry.js

```diff
@@ -0,0 +1,19 @@
+import Vue from 'vue'
+import * as Sentry from '@sentry/vue'
+import { BrowserTracing } from '@sentry/tracing'
+import router from 'src/router'
+
+Sentry.init({
+  Vue,
+  dsn: 'https://53d89028862e41a9a9b0c64df8a4c405@o1133345.ingest.sentry.io/6192706',
+  integrations: [
+    new BrowserTracing({
+      routingInstrumentation: Sentry.vueRouterInstrumentation(router())
+    //   tracePropagationTargets: ['localhost', 'my-site-url.com', /^\//]
+    })
+  ],
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production
+  tracesSampleRate: 0.25
```

Comment:
Why not 1.0 ?

Reply:
https://docs.sentry.io/platforms/javascript/performance/#verify

Sampling at a high frequency might impact the performance, even more so that most errors happens at initial rendering and there are not a lot of realtime interactions (mouse tracking, etc). 
Could be higher though.